homeassistant/components/microbees/switch.py

```diff
@@ -0,0 +1,81 @@
+"""Switch integration microBees."""
+import logging
+from typing import Any
+
+from homeassistant.components.switch import SwitchEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.exceptions import HomeAssistantError
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import BEES, CONNECTOR, COORDINATOR, DOMAIN
+from .coordinator import MicroBeesEntity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant, entry: ConfigEntry, async_add_entities: AddEntitiesCallback
+) -> None:
+    """Config entry."""
+    microbees = hass.data[DOMAIN][CONNECTOR]
+    bees = hass.data[DOMAIN][BEES]
+    coordinator = hass.data[DOMAIN][COORDINATOR]
+    switches = []
+    for bee in bees:
+        match bee.productID:
+            case 25 | 26 | 27 | 35 | 38 | 46 | 63 | 64 | 65 | 86:
+                for switch in bee.actuators:
+                    switches.append(MBSwitch(switch, bee, microbees, coordinator))
+
+    async_add_entities(switches)
+
+
+class MBSwitch(MicroBeesEntity, SwitchEntity):
+    """Representation of a microBees switch."""
+
+    def __init__(self, act, bee, microbees, coordinator) -> None:
+        """Initialize the microBees switch."""
+        self.act = act
+        self.bee = bee
+        self.microbees = microbees
+        self._state = self.act.value
+        self._attr_available = self.bee.active
+        self._coordinator = coordinator
+        self._attr_unique_id = self.act.id
```

Comment:
is the actuator id unique?

Reply:
yes it is

Reply:
but unique for a bee or over all devices?